src/assets/stylesheets/_custom.scss

```diff
@@ -73,17 +73,17 @@ $textCenter: center;
     text-overflow: ellipsis;;
 }
 @mixin transform($transform){
-	-webkit-transform: $transform;  
+	-webkit-transform: $transform;
 	-moz-transform:    $transform;
-	-ms-transform:     $transform; 
-	-o-transform:      $transform; 
+	-ms-transform:     $transform;
+	-o-transform:      $transform;
 	transform:         $transform;
 }
 @mixin transition($transition) {
-	-webkit-transition: $transition;  
+	-webkit-transition: $transition;
 	-moz-transition:    $transition;
-	-ms-transition:     $transition; 
-	-o-transition:      $transition; 
+	-ms-transition:     $transition;
```

Comment:
I think earlier versions of opera didn't support transition and that is why I thought it wouldn't work. Or it might be something else, but it appears to be valid now. That is why a caveated my original statement. :)